README.md

```diff
@@ -18,18 +18,15 @@ This is the preferred way. Does not require golang to be installed on the system
 To download a binary for the latest release, run:
 
 ```sh
-curl -sSfL https://raw.githubusercontent.com/ava-labs/camino-network-runner/main/scripts/install.sh | sh -s
+curl -sSfL https://raw.githubusercontent.com/chain4travel/camino-network-runner/chain4travel/scripts/install.sh | sh -s
 ```
 
-The binary will be installed inside the `./bin` directory (relative to where the install command was run).
-
-_Downloading binaries from the Github UI will cause permission errors on Mac._
+The binary will be installed inside the `~/bin` directory.
```

Comment:
If the installation is relative (as it is commented here and there), we should follow the $PWD thing instead sticking it to HOME

Reply:
That's because `build.sh` and `install.sh` scripts defaults to these paths, one defaults to `./bin` and one to `~/bin`.

That is how it's done at the [upstream](https://github.com/ava-labs/avalanche-network-runner) also. But I agree, we should choose one and stick to it. So, should it be the `$PWD`? If we do this I will also update `install.sh` script's default install directory.

Reply:
I've changed the scripts to use `./bin` for default path. I also need to update Camino Docs. We need some tidying there.